DataFormats/TrajectorySeed/src/classes_def.xml

```diff
@@ -13,7 +13,7 @@
   <class name="edm::RefProd<std::vector<TrajectorySeed> >"/>
   <class name="edm::RefVector<std::vector<TrajectorySeed>,TrajectorySeed,edm::refhelper::FindUsingAdvance<std::vector<TrajectorySeed>,TrajectorySeed> >"/>
   <class name="edm::RefVectorIterator<std::vector<TrajectorySeed>,TrajectorySeed,edm::refhelper::FindUsingAdvance<std::vector<TrajectorySeed>,TrajectorySeed> >"/>
-  <class pattern="std::iterator<*edm::Ref*TrajectorySeed*>"/>
+  <!--<class pattern="std::iterator<*edm::Ref*TrajectorySeed*>"/>-->
```

Comment:
it would help to have a comment inline with an explanation why this commented out line is needed